crates/uv/tests/it/build.rs

```diff
@@ -234,8 +234,8 @@ fn build() -> Result<()> {
 
     ----- stderr -----
     Building source distribution...
-    error: [TEMP_DIR]/ does not appear to be a Python project, as neither `pyproject.toml` nor `setup.py` are present in the directory
-
+      × Failed to build `[TEMP_DIR]/`
```

Comment:
I don't really know what exactly we want here... Right now, we only show the package name if they attempt to build multiple. We could also say "Failed to build source distribution for...", but it says "Building source distribution above". Or we could omit this line entirely.